react/features/salesforce/useSalesforceLinkDialog.ts

```diff
@@ -22,7 +22,8 @@ export const useSalesforceLinkDialog = () => {
     const { t } = useTranslation();
     const dispatch = useDispatch();
     const [ selectedRecord, setSelectedRecord ] = useState<{
-        id: string; name: string; onClick: Function; type: string; } | null>(null);
+        // eslint-disable-next-line no-extra-parens, @typescript-eslint/type-annotation-spacing
+        id: string; name: string; onClick:(e?: React.MouseEvent) => void; type: string; } | null>(null);
```

Comment:
could this be avoided by extracting the shape in a type/interface?
interface ISelectedRecord = {
 id: string; 
name: string; 
onClick:(e?: React.MouseEvent) => void; 
type: string; 
}